### PR TITLE
Mvc

### PR DESCRIPTION
**Pull request by Blawal-SE**

### CODE REVIEW

File: SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/AddStudent.cshtml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+
+<script>
+$(document).ready(function(){
+    $('#courseslct').select2({
+     placeholder:"choose Courses"});
+});
+
+$('#saveBtn').click(function(){
+   // debugger;
```
Comment (Nauman7) [inline SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/AddStudent.cshtml:96]: dont use ajax, please use proper MVC form post

File: SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/EditStudent.cshtml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+<script>
+    $(document).ready(function(){
+        $('#courseslct').select2();
+        $('#saveBtn').click(function () {
+                  var student = editStudent();
+                  debugger;
+                  if (validateStudentForm(student,$('#parent')))
+                  {
+                  
+                      $.ajax({
```
Comment (Nauman7) [inline SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/EditStudent.cshtml:114]: if you were using the ajax then why you have not used the generic ajax?

File: SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/Index.cshtml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+
+
+<script>
+        $(document).ready(function () {
+            $("#student").DataTable();
+            debugger;
+           // $("#messgae:contains(success)").parents('div').css("color:green;border:2px solid green");
+            if ($("#messgae:contains(Success)")) {
+                $("#messgae:contains(Success)").parents('div').css("color:green;border:2px solid green");
```
Comment (Nauman7) [inline SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/Index.cshtml:64]: this one line was enough

`$("#messgae").parents('div').css($("#messgae:contains(Success)")?"color:green;border:2px solid green":"color:red;border:2px solid red");`

File: SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/AddStudent.cshtml [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+
+
+<script>
+$(document).ready(function(){
+    $('#courseslct').select2({
+     placeholder:"choose Courses"});
+});
+
+$('#saveBtn').click(function(){
```
Comment (Shanfiaz) [inline SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/AddStudent.cshtml:95]: Try to use this ajax like a generic function used before or move it to external js file

File: SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/AddStudent.cshtml (hunk 1 above)
Comment (Shanfiaz) [inline SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/AddStudent.cshtml:96]: irrelevant comment

**Review by Shanfiaz — CHANGES_REQUESTED** [review]

File: SchoolApplication/Sxhool.WebMvc/Controllers/StudentController - Copy.cs [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        {
+            return View(CourseRepository.Instance.GetAll());
+        }
+        [HttpPost]
+        public JsonResult AddStudent(string student)
+        {
+            JsonResult result = new JsonResult();
+            result.JsonRequestBehavior = JsonRequestBehavior.AllowGet;
+            StudentDtoPost model = JsonConvert.DeserializeObject<StudentDtoPost>(student);
+            if(_repo.AddStudent(model.student,model.courses))
```
Comment (Nauman7) [inline SchoolApplication/Sxhool.WebMvc/Controllers/StudentController - Copy.cs:38]: `result.Data = new { success = _repo.AddStudent(model.student,model.courses)};`

**Review by Nauman7 — CHANGES_REQUESTED** [review]

**Pull request merged**